sakuya/mewo.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+import random
+from typing import Dict
+
+from discord import Guild
+from discord.ext import commands
+from discord.ext.commands import Bot
+
+from .db import db, Guild as _Guild, Member as _Member
+
+
+random_mewo = [
+    "ME.. oh you're not mcmower.",
+    "MEWO",
+    "You didn't really expect me to say mewo did you"
```

Comment:
style nit: add leading comma here for cleaner diffs when adding more items to the list

Reply:
Other files in the repo exclude a trailing comma, matching their style